app/src/main/java/org/mozilla/focus/home/FeatureSurveyViewHelper.kt

```diff
@@ -82,8 +84,8 @@ class FeatureSurveyViewHelper internal constructor(private val context: Context,
                     // Dismiss view immediately
                     parentView.removeView(rootView)
                     isViewInit = false
-                    // VPN app is installed, go to google play store
-                    IntentUtils.goToPlayStore(context, Constants.PACKAGE_EXPRESS_VPN)
+                    // VPN app is installed, open Play Store in a new tab
```

Comment:
I guess this comment should be "VPN app is NOT installed", so we open play store in a new tab?